packages/app/src/app/hooks/useSubscription.ts

```diff
@@ -0,0 +1,63 @@
+import {
+  SubscriptionStatus,
+  SubscriptionType,
+  TeamMemberAuthorization,
+} from 'app/graphql/types';
+import { useAppState } from 'app/overmind';
+
+export const useSubscription = () => {
+  const {
+    activeTeam,
+    activeTeamInfo,
+    personalWorkspaceId,
+    activeWorkspaceAuthorization,
+  } = useAppState();
+
+  /**
+   * Personal states
+   */
+  const isPersonalSpace = activeTeam === personalWorkspaceId;
+
+  /**
+   * Team states
+   */
+  const isTeamSpace = !isPersonalSpace;
+
+  const isTeamAdmin =
+    isTeamSpace &&
+    activeWorkspaceAuthorization === TeamMemberAuthorization.Admin;
```

Comment:
I would keep this away from the hook, unless it is needed in the subscription fields in any way. It would make the hook more focused on subscription data only and we can even have a separate `useWorkspaceAuthorization` that can return things like: `isTeamAdmin` or `isPersonalSpace`